keycloak/protocol_mapper.go

```diff
@@ -0,0 +1,10 @@
+package keycloak
+
+// https://www.keycloak.org/docs-api/4.2/rest-api/index.html#_protocolmapperrepresentation
+type protocolMapper struct {
+	Id             string                 `json:"id,omitempty"`
+	Name           string                 `json:"name"`
+	Protocol       string                 `json:"protocol"`
+	ProtocolMapper string                 `json:"protocolMapper"`
+	Config         map[string]interface{} `json:"config"`
```

Comment:
Why is the val of type `interface{}`?  It looks like it's always a string in the API responses I'm seeing.

Reply:
Haha, yeah you probably should not assume that.  The Keycloak API is all over the place with stuff like that.